### ISSUE TITLE
annotate.cells.using.markers issue

### ISSUE THREAD

**Issue by lwhitmore** [shmohammadi86/ACTIONet#28]
I keep getting the following error when I try to use annotate.cells.using.markers() function
Error in compute_marker_aggregate_stats(G = G, S = S, marker_mat = marker_mat,  : 
  internal error: spglue_merge::subview_merge(): count != merge_n_nonzero
> 

Can someone inform me what this means.  I can run the test example on the github page with no problems it seems like it could be an issue with my dataset but in some general checks it looks correct. 

**ace**
class: ACTIONetExperiment 
dim: 35432 36529 
metadata(2): ACTION_sigma backbone
assays(2): counts logcounts
rownames(35432): ENSMMUG00000023296 U6 ... ENSMMUG00000054208
  ENSMMUG00000059468
rowData names(0):
colnames(36529): AAACGAAAGCTTGTTG-1_1 AAACGAAAGTCTAGAA-1_1 ...
  TTTGTTGGTTCAACGT-1_20 TTTGTTGTCGTGTGAT-1_20
colData names(19): orig.ident nCount_RNA ... assigned_archetype
  node_centrality
rowMaps(1): unified_feature_specificity
colMaps(6): ACTION ACTIONred ACTIONet2D ACTIONet3D denovo_color archetype_footprint
rowNets(0): 
colNets(1): ACTIONet

**Comment by sebastianpineda** [comment]
Can you check that you don't have genes with all zero values in your assays? These should be removed along with any very low expression genes before running any part of the pipeline.

**Comment by lwhitmore** [comment]
Yes, I checked and I am using set of 10000 variable genes and none have all 0's.  Attached is a list of the sum of all counts of each gene across all cells in the model.  there are quite a few genes with a count of 1 though. 

Thanks for your help!
Leann
[genesums.csv](https://github.com/shmohammadi86/ACTIONet/files/8226346/genesums.csv)
e

**Comment by sebastianpineda** [comment]
That may also pose an issue. The statistics for gene enrichment can not be calculated with a single transcript. I recommend filtering all cells that have less than at least a few hundred unique genes and similarly removing all genes that are not present in at least 1% of cells.
